pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/KubernetesRuntime.java

```diff
@@ -993,6 +993,10 @@ private static String createJobName(String tenant, String namespace, String func
         return "pf-" + tenant + "-" + namespace + "-" + functionName;
     }
 
+    private static String getServiceUrl(String jobName, String jobNamespace, int instanceId) {
+        return String.format("%s-%d.%s.%s", jobName, instanceId, jobName, jobNamespace);
```

Comment:
Does there need to be `svc.cluster.local` after the namespace? https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#namespaces-and-dns

Reply:
That format worked when I tested in minikube

Reply:
maybe by default "svc.cluster.local" is added on if not specified?

Reply:
lets add it since that is what is in the documentation

Reply:
Yeah I think either should work.